src/libs/thirdparty/pikevm/re.c

```
@@ -0,0 +1,722 @@
+        INSERT_CODE(term, 2, PC);
+        EMIT(term, JMP);
+        EMIT(term + 1, REL(term, PC));
+        term = PC;
+        break;
+      } else {
+        mincnt++;
+        start0 = 1;
+      }
+      for (i = maxcnt-mincnt; i > 0; i--) {
+        EMIT(PC++, SPLIT);
+        EMIT(PC++, REL(PC, PC+((size+2)*i)));
```

cppcheck

[error] Expression 'code[prog->unilen++]=(prog->unilen+((size+2)*i)-(prog->unilen)-2)' depends on order of evaluation of side effects

(unknownEvaluationOrder)